org.codeberg.dnkl.foot.yml

```diff
@@ -0,0 +1,67 @@
+app-id: org.codeberg.dnkl.foot
+runtime: org.freedesktop.Platform
+runtime-version: "21.08"
```

Comment:
```suggestion
runtime-version: "22.08"
```

it's out now. might as well.

Reply:
The used Sdk extension didn't provide this version currently

Reply:
as I said, it's out now, might as well.

I know it wasn't available before.

Reply:
I mean `org.freedesktop.Sdk.Extension.golang` not the Freedesktop Sdk.

Reply:
the goland sdk is now build for 22.08 (will be available in a few hours)